lib-cloudsdk/src/main/java/com/facebook/openwifi/cloudsdk/UCentralUtils.java

```diff
@@ -58,6 +60,24 @@ public class UCentralUtils {
 		UCentralUtils.UPPER_CHANNEL_LIMIT.put(UCentralConstants.BAND_5G, 165);
 	}
 
+	/** List of available channels per band for use. */
+	public static final Map<String, List<Integer>> AVAILABLE_CHANNELS_BAND =
```

Comment:
I would suggest declaring this as `Collections.unmodifiableMap()` esp. if this is going into a util class. You can write some private static function for this instead of doing it in a `static` block.